### ISSUE TITLE
ESP Disconnecting Frequently

### ISSUE THREAD

**Issue by charlescbullock** [diyhue/Lights#26]
Hey guys!

So I have 7 ESP8266's - and only one of them is acting a little funky.

NOTE: I have not updated the code on the units for a since about February 18th.. I do not have the new firmware installed.

It has happened, in the past, where it connects - and after a little while - it disconnects.. Could be power surges dropping the router out of play.. Not entirely sure, as it generally happened when I'm away.

Anyways - at this point, the light is not detected at all.. once again - could definitely be router issues.

But I wanted to see if the new code does have a feature that reconnects the units, if they fail to connect at first..
Basically, whenever I power up the units now - one of them is constantly streaking red. 
Would be nice if it just simply tried to reconnect!

Once again, haven't updated yet..

Is this part of the update?

Thank you all!!

**Comment by mariusmotea** [comment]
Hi,

The ESP must automatically reconnect in case of wifi issues, i suspect the issue is hardware related and ESP remain stuck (not enough power?)
